crates/flox-rust-sdk/src/models/environment_ref.rs

```diff
@@ -209,7 +209,7 @@ impl<'flox> Named {
     }
 
     /// Return path to an owner in data dir, e.g. ~/.local/share/flox/environments/owner
-    fn owner_dir(flox: &Flox, owner: &str) -> PathBuf {
+    pub fn owner_dir(flox: &Flox, owner: &str) -> PathBuf {
```

Comment:
Is there a better place to move this to such that we can share it with `Environment`?

Reply:
Maybe we rather add a conversion for `Environment -> EnvironmentRef` and then call it as `self.as_env_ref().owner_dir()`.

Its like installable, where on the Ref its a _guessed_ value and on the Environment its a _known_ one...